README.md

```diff
@@ -35,17 +34,8 @@ Kubernetes, see:
 - [kubernetes/community#1934](https://github.com/kubernetes/community/pull/1934)
 - [kubernetes/kubernetes#64283](https://github.com/kubernetes/kubernetes/pull/64283)
 
-For the on-going work on `runc` patches upstream, please see:
-
-- [opencontainers/runc#1806](https://github.com/opencontainers/runc/pull/1806)
-- [opencontainers/runc#1688](https://github.com/opencontainers/runc/pull/1688)
```

Comment:
we need specconv library with this patch (forked into moby/buildkit/util temporary) but we don't need runc binary with this